packages/wallet/test/e2e/config.ts

```diff
@@ -31,10 +32,22 @@ export const assetProvider = (() => {
 })();
 
 export const txSubmitProvider = (() => {
-  const projectId = process.env.BLOCKFROST_API_KEY;
-  if (!projectId) throw new Error('BLOCKFROST_API_KEY not set (for txSubmitProvider)');
-  const blockfrost = new BlockFrostAPI({ isTestnet, projectId });
-  return blockfrostTxSubmitProvider(blockfrost);
+  const env = envalid.cleanEnv(process.env, {
```

Comment:
Nice

Reply:
Completed the upgrade in 2aba96af2329c4a28f078325fe9d8fc7ff3893d4